### PR TITLE
hoppet: add new package

### PR DESCRIPTION
**Pull request by haralmha**

### CODE REVIEW

**Comment by haralmha** [comment]
@vvolkl

File: var/spack/repos/builtin/packages/hoppet/package.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
```
Comment (vvolkl) [inline var/spack/repos/builtin/packages/hoppet/package.py]: >  If you submit this package back to Spack as a pull request,
>  please first remove this boilerplate and all FIXME comments.

File: var/spack/repos/builtin/packages/hoppet/package.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+from spack import *
+
+
+class Hoppet(AutotoolsPackage):
+    """A Fortran 95 package for carrying out QCD DGLAP evolution and other common manipulations of parton distribution functions (PDFs)."""
+
+    homepage = "https://hoppet.hepforge.org/"
+    url      = "https://github.com/gavinsalam/hoppet/archive/refs/tags/hoppet-1.2.0.tar.gz"
+
+
```
Comment (vvolkl) [inline var/spack/repos/builtin/packages/hoppet/package.py]: please add a hep tag, like:
 `tags = ['hep']`

File: var/spack/repos/builtin/packages/hoppet/package.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    # maintainers = ['haralmha']
```
Comment (vvolkl) [inline var/spack/repos/builtin/packages/hoppet/package.py]: If you are ok with being pinged for reviews for this recipe, uncomment, if not, remove this line

File: var/spack/repos/builtin/packages/hoppet/package.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    version('1.2.0', sha256='6e00eb56a4f922d03dfceba7b389a3aaf51f277afa46d7b634d661e0797e8898')
+    
+
+    def configure_args(self):
+        # FIXME: Add arguments other than --prefix
+        # FIXME: If not needed delete this function
```
Comment (vvolkl) [inline var/spack/repos/builtin/packages/hoppet/package.py]: > FIXME: If not needed delete this function

**Comment by vvolkl** [comment]
There are issues with the code style, see https://spack.readthedocs.io/en/latest/contribution_guide.html#style-tests to fix them.

**Review by vvolkl — CHANGES_REQUESTED** [review]
See above. Also, can you change the title? Most PRs use a format similar to 
"hoppet: add new package"
and
"hoppet: add version xyz"

Did you try to run the tests with `spack install --test=root hoppet`?

**Comment by vvolkl** [comment]
Your first commit does not seem to have proper author information, maybe rebase to squash all of them?

**Review by vvolkl — APPROVED** [review]

**Review by alalazo — APPROVED** [review]

**Pull request merged**